scripts/add_extra_files_to_editor.sh

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env bash
+
+command -v jq >/dev/null 2>&1 || {
+        echo >&2 ">>>>> This script requires jq but it's not installed. Aborting."
+        exit 1
+}
+
+#######################################################
+################### HELP ##############################
+print_usage() {
+        echo -e "\nDescription:\n"
+        echo -e "This script goes through each exercise directory, picks all the java files in the src directory, except the main editable solution file, and adds them to 'fonts.editor' section of the config.json file present in the .meta folder of the exercise" 
+        echo -e "This makes sure that there is no need to manually configure the config.json for subsequent exercises. Just running this script will resolve all those tasks on their own."
+
+        echo -e "\nSyntax: ./add_extra_files_to_editor.sh [-a|-h|-d <exercise_name>]"
+
+        echo -e "\nOptions:\n"
+        echo -e "h                      Display the usage of the script"
+        echo -e "a                      Processes all the exercises present inside ./exercise/practice directory"
+        echo -e "d <exercise_name>      Processes only the directory specified by the argument <exercise_name>." 
+        echo -e "                       Do note that you only need to give the name of the exercise. Do not give the path to the exercise folder."
+        echo -e "                       For example: if you need to process only the exercise 'triangle' then type '<script_name>.sh -d triangle'"
+}
+
+#######################################################
+#######################################################
+
+
+#######################################################
+################### MAIN ##############################
+
+
+# get the current location
+current_location=$PWD
+
+# get the exercises location
+exercises_location="$current_location/exercises/practice"
+
+process_current_directory() {
+
+        java_source_dir="$exercises_location/$1/src/main/java"
+
+        data=$(find "$java_source_dir" -type f -name "*.java")
+
+        # now getting the config.json file and here getting the details of the solution file
+        json_file="$exercises_location/$1/.meta/config.json"
+
+        # this is the solution file here
+        solution_file=$(jq '.| .files.solution[0]' $json_file)
+        solution_file=${solution_file:1:-1} # removing the quotes
```

Comment:
```suggestion
        solution_file=$(jq -r '.files.solution[0]' $json_file)
```

The `-r` flag means "raw output" and makes jq output strings without quotes. Also `${solution_file:1:-1}` was not working on my machine with an `line 50: -1: substring expression < 0` error.

Also, it'd be great if the script could take into account all solution files, and not just the first one. For instance, the exercise [error-handling](https://github.com/exercism/java/tree/main/exercises/practice/error-handling) has 3 files as solution files. The current version of the script adds 2 of them to the `editor` key, when it shouldn't.